aws/resource_aws_glue_job.go

```diff
@@ -113,6 +114,18 @@ func resourceAwsGlueJob() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+			"worker_type": {
+				Type:          schema.TypeString,
+				Optional:      true,
+				ConflictsWith: []string{"allocated_capacity", "max_capacity", "allocated_capacity"},
+				ValidateFunc:  validation.StringMatch(regexp.MustCompile(`^Standard|G.1X|G.2X`), "Must be one of Standard, G.1X or G.2X. See https://docs.aws.amazon.com/glue/latest/dg/aws-glue-api-jobs-job.html"),
```

Comment:
Nit: We typically prefer to use the [`validation.StringInSlice()`](https://godoc.org/github.com/hashicorp/terraform-plugin-sdk/helper/validation#StringInSlice) functionality coupled with available [AWS Go SDK constants](https://docs.aws.amazon.com/sdk-for-go/api/service/glue/#pkg-constants) for this type of validation, to limit the usage of regular expressions while also setting us up for future auto-generation opportunities, e.g.

```suggestion
				ValidateFunc:  validation.StringInSlice([]string{
					glue.WorkerTypeG1x,
					glue.WorkerTypeG2x,
					glue.WorkerTypeStandard,
				}, false),
```

Reply:
This is not formatted through `make fmt`. There is two spaces vetween `ValidateFunc:` and `validation`. It's currently making all PR tests fail.

Reply:
I think I ran `go fmt` and not `make fmt` from the root per the CONTRIBUTING.md's Enhancement/Bugfix to a Resource -> Well-formed Code section. I can run `make fmt` and push to the PR.

Reply:
No problem, fixed by 4e449b0

Reply:
thank you, sorry for an inconvenience!